spec/request/target/index_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+describe 'GET #index target', type: :request do  
+  let!(:user)     { create(:user) }
+  let!(:targets)  { create_list(:target, 2, user: user) }
+
+  subject do
+    get api_v1_targets_path, headers: auth_headers, as: :json
+  end
+
+  before { subject }
+
+  context 'when valid' do
+    let!(:auth_headers) { auth_user_headers }
+
+    it 'gets a successfull answer' do
+      expect(response).to be_successful
+    end
+
+    it 'returns list of targets' do
+      expect(parsed_data['targets'].count).to eq(targets.count)
+    end
```

Comment:
The title of the spec is incorrect here, you are testing the correct amount of target.
I'd also test that the `ids` are the same as expected using `match_array `